src/main/scala/exercises/06_Collections.scala

```diff
@@ -30,16 +30,12 @@ object Collections {
    * Hint: there is a built-in function for this you can use.
    * 
    */
-  def firstElementInList[T](l: List[T]): T = {
-    error("fix me")
-  }
+  def firstElementInList[T](l: List[T]): T = l(0)
 
   /**
    * Get the sum of all the elements in the list, e.g. sumOfList(List(1,2,3)) = 6.
    */
-  def sumOfList(l: List[Int]): Int = {
-    error("fix me")
-  }
+  def sumOfList(l: List[Int]): Int = (0 /: l)(_+_)
```

Comment:
Woah, I hadn't seen that syntax before! Is that like an infix version of `fold`?